src/main/java/com/sonyericsson/jenkins/plugins/bfa/model/indication/FoundIndication.java

```diff
@@ -58,6 +59,7 @@
     private String pattern;
     private Run build;
```

Comment:
Is this supposed to be here?  Doesn't this end up creating a cycle that makes collection difficult?